lib/r10k/module/git.rb

```diff
@@ -23,6 +23,11 @@ def self.implement?(name, args)
   #   @return [String]
   attr_reader :desired_ref
 
+  # @!attribute [r] default_ref
+  #   @api private
+  #   @return [String]
+  attr_reader :default_ref
```

Comment:
Where does this get set?

Reply:
Oh I see it. Nevermind. We just were never retrieving it form the outside before, and now we are in the test. Sometimes I hate how Ruby doesn't make you declare instance variables anywhere centralized...